### PR TITLE
[servus] Updated extractor to work with new website

### PR DESCRIPTION
**Pull request by chfoidl**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [X] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [X] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [X] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [X] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [ ] Improvement
- [X] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Updated the existing `servus` extractor to work with the updated websites of servustv.com and pm-wissen.com.

### CODE REVIEW

File: yt_dlp/extractor/servus.py [hunk 1]
```diff
@@ -17,128 +13,73 @@ class ServusIE(InfoExtractor):
...
-        attrs = {}
-        for attribute in video['attributes']:
-            if not isinstance(attribute, dict):
-                continue
-            key = attribute.get('fieldKey')
-            value = attribute.get('fieldValue')
-            if not key or not value:
-                continue
-            attrs[key] = value
+        formats = self._extract_m3u8_formats(
```
Comment (FrankZ85) [inline yt_dlp/extractor/servus.py:56]: Please use self._extract_m3u8_formats_and_subtitles here.
https://www.servustv.com/natur/v/aansszcx3yi9jmlmhdc1/ is one example of a link with subtitles.

File: yt_dlp/extractor/servus.py [hunk 2]
```diff
@@ -17,128 +13,73 @@ class ServusIE(InfoExtractor):
...
-        season = attrs.get('season')
-        episode = attrs.get('chapter')
-        duration = float_or_none(attrs.get('duration'), scale=1000)
+        title = video['title']
+        description = video['description']
+        thumbnail = video['poster']
+        series = video['label']
+        season = video['season']
+        episode = video['chapter']
+        duration = float_or_none(video['duration'], scale=1000)
```
Comment (FrankZ85) [inline yt_dlp/extractor/servus.py:71]: scale=1000 is probably wrong. The value is in seconds.

File: yt_dlp/extractor/servus.py [hunk 3]
```diff
@@ -17,128 +13,73 @@ class ServusIE(InfoExtractor):
...
             r'Episode (\d+)', episode or '', 'episode number', default=None))
 
         return {
             'id': video_id,
             'title': title,
-            'alt_title': alt_title,
             'description': description,
             'thumbnail': thumbnail,
             'duration': duration,
-            'timestamp': unified_timestamp(video.get('lastPublished')),
```
Comment (FrankZ85) [inline yt_dlp/extractor/servus.py:141]: I think the timestamp is in "currentSunrise".

**Comment by pukkandan** [comment]
@FrankZ85 The PR author hasn't responded in months. I doubt there will be any use from your review. I recommend making a new PR if you can

**Pull request closed without merging**